### DISCUSSION TITLE
Render info only showing time now: where did used VRAM etc go?

### DISCUSSION THREAD

**Discussion by Tillerz** [vladmandic/sdnext#665] [Q&A] [answered]
- Render info only showing time now: where did used VRAM etc go? 

`Time** taken: 1m 1.30s`

That's all the output now. Where is the x/y VRAM use etc we had before?

- Also I just saw that the settings for rendering on the left show "Clip skip 1", but the render log at the right shows 2 (after moving the slider somewhat and then back to 1 the next render did clip skip 1).

- One more: when loading the list of checkpoints, it's looking for a checkpoint "None" now:

```
Checkpoint not found: None
Available models: X:\automatic\models\Stable-diffusion 56
```

**Comment by vladmandic** [comment, marked as answer]
fixed, wait for ~1h for github push.
